src/test/javascript/spec/component/modeling-editor/modeling-editor.component.spec.ts

```diff
@@ -151,4 +150,82 @@ describe('ModelingEditorComponent', () => {
         const umlElement = fixture.componentInstance.elementWithMethod('method', model);
         expect(umlElement).toBeTruthy();
```

Comment:
As you did the sinon/chai conversion in this file: Can you please replace the various `toBeTruthy()`/`toBeFalsy()` calls by `.not.toBe(null)`/`.toBe(null)` for elements queried By CSS, or a concrete assertion about the value if possible/reasonable otherwise?